packages/bruno-cli/src/utils/common.js

```diff
@@ -14,7 +19,32 @@ const rpad = (str, width) => {
   return paddedStr;
 };
 
+const createFormData = (datas, collectionPath) => {
+  // make axios work in node using form data
+  // reference: https://github.com/axios/axios/issues/1006#issuecomment-320165427
+  const form = new FormData();
+  forOwn(datas, (value, key) => {
+    if (typeof value == 'object') {
```

Comment:
I would personally invert the `if` to: `typeof value === 'string'` and return after `form.append` like this:

```js
if (typeof data === 'string') {
  form.append(key, value);
  return;
}

// Code for file here.
```
To make the code more readable